jnosql-mapping/jnosql-mapping-graph/pom.xml

```diff
@@ -31,15 +31,18 @@
         <neo4j.connector.version>0.9-3.4.0</neo4j.connector.version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>org.eclipse.jnosql.mapping</groupId>
+            <artifactId>jnosql-mapping-core</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.eclipse.jnosql.communication</groupId>
             <artifactId>jnosql-communication-query</artifactId>
-            <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>jnosql-mapping-core</artifactId>
-            <version>${project.version}</version>
+            <groupId>jakarta.nosql.tck.mapping</groupId>
```

Comment:
@dearrudam, the TCK does not exist anymore, at least yet.

Reply:
@otaviojava could I remove the TCK dependencies from the `dependencyManagement` tag in the `jnosql-parent`'s POM?

Reply:
Yes, it does not exist anymore.

Reply:
@otaviojava okay! thanks! I'm going to remove it!